### ISSUE TITLE
ACL filter - 'list' object has no attribute 'values

### ISSUE THREAD

**Issue by lvrfrc87** [lvrfrc87/pyateos-ansible#2]
### BUG

When more than 1 ACL in running config

```
 "module_stderr": "Traceback (most recent call last):
\n  File \"/var/lib/awx/.ansible/tmp/ansible-local-1155lcny70s0/ansible-tmp-1598456544.71066-141832271525844/AnsiballZ_eos_pyateos.py\", line 102, in <module>
_ansiballz_main()
\n  File \"/var/lib/awx/.ansible/tmp/ansible-local-1155lcny70s0/ansible-tmp-1598456544.71066-141832271525844/AnsiballZ_eos_pyateos.py\", line 94, in _ansiballz_main
\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
\n  File \"/var/lib/awx/.ansible/tmp/ansible-local-1155lcny70s0/ansible-tmp-1598456544.71066-141832271525844/AnsiballZ_eos_pyateos.py\", line 40, in invoke_module
\n    runpy.run_module(mod_name='ansible.modules.eos_pyateos', init_globals=None, run_name='__main__', alter_sys=True)
\n  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module
\n    return _run_module_code(code, init_globals, run_name, mod_spec)
\n  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code
\n    mod_name, mod_spec, pkg_name, script_name)
\n  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code
\n    exec(code, run_globals)
\n  File \"/tmp/ansible_eos_pyateos_payload_1uih_cv2/ansible_eos_pyateos_payload.zip/ansible/modules/eos_pyateos.py\", line 600, in <module>
\n  File \"/tmp/ansible_eos_pyateos_payload_1uih_cv2/ansible_eos_pyateos_payload.zip/ansible/modules/eos_pyateos.py\", line 594, in main
\n  File \"/tmp/ansible_eos_pyateos_payload_1uih_cv2/ansible_eos_pyateos_payload.zip/ansible/modules/eos_pyateos.py\", line 452, in run_compare
\n  File \"/tmp/ansible_eos_pyateos_payload_1uih_cv2/ansible_eos_pyateos_payload.zip/ansible/modules/eos_pyateos.py\", line 311, in filter_jmespath\
n  File \"/tmp/ansible_eos_pyateos_payload_1uih_cv2/ansible_eos_pyateos_payload.zip/ansible/modules/eos_pyateos.py\", line 353, in filter_acls_counters
\nAttributeError: 'list' object has no attribute 'values'\n",
```

**Comment by lvrfrc87** [comment]
```
       "insert": [
            [
                1,
                {
                    "readonly": False,
                    "countersIncomplete": False,
                    "name": "TEST",
                    "staleCounters": False,
                    "sequence": [
                        {
                            "ruleFilter": {
                                "nvgre": {
```

**Comment by lvrfrc87** [comment]
FIx in 1.0.6
